### ISSUE TITLE
[Bug] window.open does not open a new window/tab

### ISSUE THREAD

**Issue by denschub** [mozilla-mobile/fenix#3702]
## Steps to reproduce

1. Open a site that attempts to open a popup via `window.open`. I provided an example [here](https://stuff.schub.io/mozilla/testcases/webc-32144/).
2. Click the button and accept the popup warning.

### Expected behavior

* A new tab loading mozilla.org is loaded.

### Actual behavior

* mozilla.org is loading in the same tab, navigating away from the test case.

### Additional information

This is related to [bug 1557510](https://bugzilla.mozilla.org/show_bug.cgi?id=1557510), where the same issue came up in the Reference Browser because it made some tests fail.

We discovered this while debugging [reports about "Sign in with Google" buttons not working in Fenix](https://github.com/webcompat/web-bugs/issues/32144), and it turns out they depend on opening the authentication page in a new tab and sending messages back to the opener, as well as closing it. I am absolutely sure there are many other critical issues in the web because of this. So, uh, 🔥.

**Comment by miketaylr** [comment]
cc @vesta0

**Comment by cpeterso** [comment]
Note that the GV bug 1557510 is fixed in GV 69. Fenix just needs to update to GV 69.

**Comment by bifleming** [comment]
Waiting on either: 1) moving to GV 69 or 2) uplifting fix to 68.

**Comment by cpeterso** [comment]
This is GV bug https://bugzilla.mozilla.org/show_bug.cgi?id=1510314, which @snorp just fixed in GV 70 Nightly.

GV bug 1557510 linked above is a red herring. It only fixed the bug in the geckoview_example test app, which won't fix anything in Fenix!

**Comment by bifleming** [comment]
Waiting on GV 70. @cpeterso this is a P1 bug for Fenix -- what are the changes of uplifting to GV 69?

**Comment by cpeterso** [comment]
> Waiting on GV 70. @cpeterso this is a P1 bug for Fenix -- what are the changes of uplifting to GV 69?

I'll ask snorp. He wasn't comfortable uplifting the somewhat risky fix to a GV 68.0.x dot release, but GV 69 Beta might be OK since Christian tested the fix in GV 70 Nighty:

https://bugzilla.mozilla.org/show_bug.cgi?id=1510314#c20

**Comment by bifleming** [comment]
Waiting for uplift to GV 69

**Comment by cpeterso** [comment]
The uplift to GV 69 Beta was approved, but snorp needs to fix a merge conflict to complete the uplift:

https://bugzilla.mozilla.org/show_bug.cgi?id=1510314#c23

**Comment by agrastiOs** [comment]
It was decided to wait for GV 70 instead.

**Comment by liuche** [comment]
Moving this to bug backlog since we're now on GV 70.

**Comment by softvision-miralobontiu** [comment]
Verified as fixed with the latest Nightly build 190909 (#12521817) with Motorola Moto G6 (Android 8), and Huawei P9 Lite (Android 6).
